lib/api/attributes.js

```diff
@@ -125,6 +126,13 @@ exports.prop = function (name, value) {
       case 'nodeName':
         property = this[0].name.toUpperCase();
         break;
+      case 'href':
+      case 'src':
+        property = getProp(this[0], name);
```

Comment:
Also, move this into the `if` block?

Reply:
what mean for into if?
u mean move this code to case default and make a if block?

but i think is no need

the default is `property = getProp(this[0], name);`

and when we didn't set baseurl
also is  `property = getProp(this[0], name);`